src/main/io/gps.c

```diff
@@ -327,8 +326,7 @@ static void gpsSetState(gpsState_e state)
 
 void gpsInit(void)
 {
-    gpsSampleRateHz = 0.0f;
-
+    gpsDataIntervalSeconds = 0.1f;
```

Comment:
```suggestion
    gpsDataIntervalSeconds = 0.0f;
```
At initialization we don't know the sample rate yet.

Reply:
@KarateBrot On the other hand, this avoids possible divide by zero issues, and we do divide something by this value, so we should avoid zero.  Also 0.1f is the default rate that we request from GPS units.

Reply:
Alright, makes sense